crates/red_knot_python_semantic/resources/mdtest/subscript/string.md

```diff
@@ -20,11 +23,10 @@ reveal_type(b)  # revealed: Unknown
 ## Function return
 
 ```py
-def add(x: int, y: int) -> int:
-    return x + y
```

Comment:
I am assuming `add` was intended to be a function that definitely returns an `int`, not a `LiteralInt`. Elsewhere, we just use `def int_instance() -> int: ...` for this. Let me know if I'm misunderstanding the intention.